Code/GraphMol/SmilesParse/CXSmilesOps.cpp

```diff
@@ -2049,11 +2049,80 @@ std::string get_bond_config_block(
     bool isAnAtropisomer = false;
 
     const Atom *firstAtom = bond->getBeginAtom();
-    for (auto bondNbr : mol.atomBonds(firstAtom)) {
-      if (bondNbr->getStereo() == Bond::BondStereo::STEREOATROPCW ||
-          bondNbr->getStereo() == Bond::BondStereo::STEREOATROPCCW) {
-        isAnAtropisomer = true;
-        break;
+    if (bd == Bond::BondDir::BEGINDASH || bd == Bond::BondDir::BEGINWEDGE) {
+      for (auto bondNbr : mol.atomBonds(firstAtom)) {
+        if (bondNbr->getIdx() == bond->getIdx()) {
+          continue;  // a bond is not its own neighbor
+        }
+        if (bondNbr->getStereo() == Bond::BondStereo::STEREOATROPCW ||
+            bondNbr->getStereo() == Bond::BondStereo::STEREOATROPCCW) {
+          isAnAtropisomer = true;
+
+          // if it is for an atropisomer and there are no coords, check to see
+          // if the wedge needs to be flipped based on the smiles reordering
+          if (!coordsIncluded && isAnAtropisomer) {
+            Atropisomers::AtropAtomAndBondVec atomAndBondVecs[2];
+            if (!Atropisomers::getAtropisomerAtomsAndBonds(
+                    bondNbr, atomAndBondVecs, mol)) {
+              throw RDKit::SmilesParseException(
```

Comment:
This shouldn't be a `SmilesParseException` since we aren't parsing SMILES at this point.
This should either be a `ValueError` if it's something which could happen but shouldn't or handled using the RDKit's `CHECK_INVARIANT()` if this only happens in the case of an RDKit bug.

Reply:
changed to value error